dxr/plugins/rust/__init__.py

```diff
@@ -436,8 +436,13 @@ def process_csv(self, file_name, header_only):
                         print " - 'process_" + line[0] + "' not implemented!"
                         continue
 
-                    if 'file_name' in args and args['file_name'].startswith(self.tree.source_folder):
-                        args['file_name'] = args['file_name'][len(self.tree.source_folder)+1:]
+                    if 'file_name' in args:
+                        file_name_value = os.path.normpath(args['file_name'])
```

Comment:
Did you consider https://docs.python.org/2/library/os.path.html#os.path.relpath? I think it does all this for you.